cirq-core/cirq/circuits/circuit.py

```diff
@@ -62,7 +62,7 @@
 from cirq.circuits.qasm_output import QasmOutput
 from cirq.circuits.text_diagram_drawer import TextDiagramDrawer
 from cirq.protocols import circuit_diagram_info_protocol
-from cirq.type_workarounds import NotImplementedType
+from types import NotImplementedType
```

Comment:
Please move under the line 27 so this import is in the builtin-imports section and preserves their alphabetic order.
Same for the other sources below.

Reply:
I've made the update.

I'll address the CI-reported errors now

Reply:
@pavoljuhas I have addressed the CI-reported errors